### DISCUSSION TITLE
有辦法調整youtube譯文的高度嗎？

### DISCUSSION THREAD

**Discussion by babyhow** [immersive-translate/immersive-translate#2129] [Help]
### 概述 | Summary

我通常都選擇只顯示譯文
但無論選擇譯文顯示在上下方都還是顯示在影片最底層
看起來有點不順眼

有其他地方可設定嗎？

**Comment by theowenyoung** [comment]
可以截图演示一下吗？

**Reply by babyhow** [reply]
![2024-09-28_134058](https://github.com/user-attachments/assets/072f40ed-f1fa-4954-9aca-41dede667c30)

有可能譯文再往上調高一點嗎？

**Reply by theowenyoung** [reply]
这个应该可以直接拖动，后台会自动记住你拖动的位置。
